pkg/master/controller.go

```diff
@@ -40,14 +41,17 @@ import (
 )
 
 const kubernetesServiceName = "kubernetes"
+const clusterInfoConfigMapName = "cluster-info"
+const clusterID = "cluster-id"
```

Comment:
@mtaufen Is this satisfactory as is, or does it need to be adjusted to `alpha-cluster-id` per https://github.com/kubernetes/community/pull/662#issuecomment-309445063 ?